backend/typescript/rest/authRoutes.ts

```diff
@@ -135,18 +136,14 @@ authRouter.post(
 );
 
 /* Emails a password reset link to the user with the specified email */
-authRouter.post(
-  "/resetPassword/:email",
-  isAuthorizedByEmail("email"),
-  async (req, res) => {
-    try {
-      await authService.resetPassword(req.params.email);
-      res.status(204).send();
-    } catch (error: unknown) {
-      res.status(500).json({ error: getErrorMessage(error) });
-    }
-  },
-);
+authRouter.post("/resetPassword/:email", async (req, res) => {
```

Comment:
we remove the `isAuthorizedByEmail` since one way of resetting password is with a **non-logged in user**. Additionally, the Firebase API service will not reset successfully if the email is not verified in our database.